scenarios/csharp/dotnet/captioning/Program.cs

```diff
@@ -0,0 +1,417 @@
+/*
+Dependencies
+*/
+
+using System; // File
+using System.Linq; // Enumerable.Contains
+using System.Text; // StringBuilder
+using System.Threading.Tasks; // Task
+
+/* Notes:
+- Install the Speech NuGet package:
+dotnet add package Microsoft.CognitiveServices.Speech
+- The Speech SDK on Windows requires the Microsoft Visual C++ Redistributable for Visual Studio 2019 on the system. See:
+https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-sdk?tabs=windows%2Cubuntu%2Cios-xcode%2Cmac-xcode%2Candroid-studio
+*/
+using Microsoft.CognitiveServices.Speech;
+using Microsoft.CognitiveServices.Speech.Audio;
+
+namespace Caption
```

Comment:
It doesn't get more nitpicky than this, but I'd suggest "Captions" or "Captioning" for the namespace -- probably "Captioning" given the folder structure